### PR TITLE
User login

### PR DESCRIPTION
**Pull request by Primdahl26**
Our system is based on people being logged in (unless we can do smth smart)

### CODE REVIEW

File: frontend/src/routes/login.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+      body: payload,
+    })
+      .then((response) => response.json())
+      .then((data) => (loginStatus = data.status))
+
+    if (loginStatus === "success") {
+      alert("hooray you signed in")
```
Comment (AndreasPB) [inline frontend/src/routes/login.svelte]: We should probably do something with window.location or window.history and redirect the user somewhere
```suggestion
    if (loginStatus === "success") {
      alert("hooray you signed in")
      window.location.href = "/"
```

File: frontend/src/routes/login.svelte [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    />
+  </div>
+  <div class="flex justify-center pt-5">
+    {#if username && password}
+      <button class="btn btn-primary" on:click={signIn}>Sign in</button>
+    {:else}
+      <button class="btn btn-primary" disabled={true}>Sign in</button>
+    {/if}
+  </div>
+</div>
```
Comment (AndreasPB) [inline frontend/src/routes/login.svelte]: To make the user able to submit with return/enter it needs to be in a form, and the button should have type="submit"

```suggestion
<div class="p-10 card bg-base-200">
  <form>
    <div class="form-control">
      <label class="label">
        <span class="label-text">Username</span>
      </label>
      <input
        type="text"
        placeholder="username"
        class="input"
        bind:value={username}
      />
      <label class="label">
        <span class="label-text">Password</span>
      </label>
      <input
        type="text"
        placeholder="password"
        class="input"
        bind:value={password}
      />
    </div>
    <div class="flex justify-center pt-5">
      {#if username && password}
        <button class="btn btn-primary" type="submit" on:click={signIn}>Sign in</button>
      {:else}
        <button class="btn btn-primary" disabled={true}>Sign in</button>
      {/if}
    </div>
    <!-- <button type="submit">Submit</button> -->
  </form>
</div>
```

**Review by AndreasPB — APPROVED** [review]
LGTM with the changes

If you want it would be nice to get rid of the A11y warnings

File: frontend/src/routes/login.svelte (hunk 1 above)
Comment (Primdahl26) [reply frontend/src/routes/login.svelte]: Yeah obviously the alert is also just a temp as we dont have anywhere to redirect really :P

**Comment by AndreasPB** [comment]
Feels really nice to use with all the UX feedback. In the future it could be cool to redirect the user instantly(instead of the 250ms hardcoded setTimeout) and build the `Hooray you signed in!` at the bottom or something like that

**Comment by AndreasPB** [comment]
If this PR is for the UI and the mock functionality, we should create a few issues for the next steps - Like building the UI based on who is logged in, and the whole database coupling part

**Review by AndreasPB — CHANGES_REQUESTED** [review]
I don't know if we should do more with this before merging - If the authorization part isn't being done here it should be it's own issue

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**